pkg/collector/stats/utils.go

```diff
@@ -224,8 +224,16 @@ func getS390xArchitecture() (string, error) {
 func getCPUMicroArchitecture(family, model, stepping string) (string, error) {
 	yamlBytes, err := os.ReadFile(CPUModelDataPath)
 	if err != nil {
-		klog.Errorf("failed to read cpus.yaml: %v", err)
-		return "", err
+		currentFilePath, err := os.Getwd()
+		if err != nil {
+			return "", err
+		}
+		LocalCPUModelDataPath := strings.SplitAfter(currentFilePath, "kepler")[0]
```

Comment:
```suggestion
		localCPUModelDataPath := strings.SplitAfter(currentFilePath, "kepler")[0]
```
Because an uppercase variable name signals that this is part of the public API of a module